apps/chat/components/ui/breadcrumb.tsx

```diff
@@ -0,0 +1,109 @@
+import { Slot } from '@radix-ui/react-slot'
```

Comment:
for each ui component here, try whether it also works with its design system counterpart. if yes or with minimal customization, remove it from here to reduce code duplication